spring-boot-admin-starter-client/src/main/java/de/codecentric/boot/admin/client/config/AdminProperties.java

```diff
@@ -36,6 +36,16 @@
 	private long period = 10_000L;
 
 	/**
+	 * Connect timeout (in ms) for the registration.
+	 */
+	private int connectTimeout = -1;
```

Comment:
I would prefer another default, but this should maintain the existing behaviour...